x-pack/plugin/sql/sql-action/src/main/java/org/elasticsearch/xpack/sql/action/SqlQueryResponse.java

```diff
@@ -173,7 +175,12 @@ public static XContentBuilder value(XContentBuilder builder, Mode mode, Object v
             ZonedDateTime zdt = (ZonedDateTime) value;
             // use the ISO format
             builder.value(StringUtils.toString(zdt));
-        } else {
+        } else if (mode == CLI && value != null && value.getClass().getSuperclass().getSimpleName().equals(INTERVAL_CLASS_NAME)) {
```

Comment:
I think it's better to use the `isAssignableFrom()` and avoid the string check.

Reply:
Agree, but `Interval` is not visible in the `action` project.

Reply:
ah ok, thx!